.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 defaults: &defaults
   working_directory: ~/clark-gateway
   docker:
-    - image: cyber4all/circleci-aws:node-alpine
```

Comment:
After these go in are these images done?

Reply:
By done, do you mean not used in any of our configs?

Reply:
Yes